BruteShark/PcapAnalyzer/Analyzer.cs

```diff
@@ -11,7 +11,7 @@ public class Analyzer
 
         public event EventHandler<ParsedItemDetectedEventArgs> ParsedItemDetected;
 
-        public List<string> AvailableModulesNames => _availbleModules.Select(m => m.Name).ToList();
+        public List<string> AvailableModulesNames => _availbleModules.Select(m => m.Name).ToList().Concat(_availbleModules.Select(m => m.CliName).ToList()).ToList();
```

Comment:
The mapping between the "CLI Names" and "Analyzer Names" should be in the PL layer (In the future class "SingleCommandRunner") since the BLL should not know the PL and since it is relevant only to the single command mode